src/lib/autodiscovery/gcp_metrics_descriptor.py

```diff
@@ -85,10 +87,10 @@ def _cast_metric_kind_to_dt_format(metric_kind: str, value_type: str) -> str:
         elif metric_kind == "CUMULATIVE":
             return "count,delta"
         else:
```

Comment:
As I told before: no need for this check. You can only keep if/elif and raise exception at the end

Reply:
I thing both ways are good: https://peps.python.org/pep-0008/#programming-recommendations
![image](https://github.com/dynatrace-oss/dynatrace-gcp-monitor/assets/138470854/fa4613aa-f7bc-4d01-9fce-ac4f5f6b54db)

In this codebase it's usually to have with else approach.
Even with Your code 😄:
https://github.com/dynatrace-oss/dynatrace-gcp-monitor/commit/7c0952d9cb37cce30500c9523574aff84306a2d7#diff-9f71d84264b0b7aae201479a9cb46ca9edfd78dea0b37687ffcbcf28f070c58fR64
I duno what version in "better" 😉